CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking changes
 
 ### Bugfixes
-* Set cache headers for 404 and 400 to 0. [#649](https://github.com/elastic/package-registry/pull/649)
+* Set cache headers for 404 and 400 to private, no-store. [#649](https://github.com/elastic/package-registry/pull/649)[#652](https://github.com/elastic/package-registry/pull/652)
```

Comment:
hmm we might want to leave this as it was and move the `652` line to unreleased right? Since we technically already released 0.13.0 🤷

Reply:
ups, yes :-(

Reply:
Updated. I kept the correction to the old changelog in as it was not accurate.